.ci/pipeline/ci.yml

```diff
@@ -134,6 +134,9 @@ jobs:
       Python3.13_Sklearn1.5:
         PYTHON_VERSION: '3.13'
         SKLEARN_VERSION: '1.5'
+      Python3.13_Sklearn1.6:
```

Comment:
Please remove one of the other non-supported versions. Azure pipelines is already way overloaded with 3 hour runtimes.

Reply:
I reduced matrix to last 5 versions of python and sklearn.